options/locale/locale_fr-FR.ini

```diff
@@ -309,7 +309,7 @@ password_change_disabled=Les utilisateurs non-locaux n'ont pas le droit de modif
 emails=Adresses e-mail
 manage_emails=Gérer les adresses e-mail
 email_desc=Votre adresse e-mail principale sera utilisée pour les notifications et d'autres opérations.
-primary=Principale
+primary=Principalef
```

Comment:
Probably a typo.
Futhermore, only  en-US should be updated the rest will updated via crowdin.com